From 9b13cb12973fac422423eec1c6a91f21b5257c92 Mon Sep 17 00:00:00 2001
From: David Teigland <teigland@redhat.com>
Date: Fri, 3 Aug 2012 14:24:07 -0500
Subject: [PATCH] sanlock: remove umask 0

Remove umask(0) which causes sanlock.log to have mode 666.
It's 644 without the umask.

Signed-off-by: David Teigland <teigland@redhat.com>
---
 src/main.c |    1 -
 1 file changed, 1 deletion(-)

--- a/src/main.c
+++ b/src/main.c
@@ -1198,7 +1198,6 @@
 			log_tool("cannot fork daemon\n");
 			exit(EXIT_FAILURE);
 		}
-		umask(0);
 	}
 
 	/* main task never does disk io, so we don't really need to set
